src/blr/cli/clusterrmdup.py

```diff
@@ -77,8 +76,6 @@ def main(args):
                 # New chr => reset dup pos cache and rp cache tracker
                 else:
                     cache_dup_pos = dict()
-                    for cache_read_pair_tracker in current_cache_rp.values():
-                        summary.reads_at_analyzed_dup_position += len(cache_read_pair_tracker.current_reads) * 2
```

Comment:
Why was this removed? Should you not analyze the duplicates for the last position on the previous chromosome?

Reply:
Turns out it could be corrected and simplified with the commit last commit.